.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+## Overview
+<!--  
+Provide a high-level description of this change.   
+-->
+
+
+
+## Type of change
+<!--  
+Check the box below that describes your change best:
+--> 
+
+- [ ] new plugin
+- [ ] improvement for an existing plugin
+- [ ] bug-fix for an existing plugin
+- [ ] changes in contributor's experience
```

Comment:
How about something like:
```suggestion
- [ ] Created a new plugin
- [ ] Improved an existing plugin
- [ ] Fixed a bug in an existing plugin
- [ ] Improved contributor utilities or experience
```

And will we be using these in the actual changelog?

Reply:
Yes, these will be used in order to determine the changelog type: new, improved or fixed.